### PR TITLE
[chore] update contrib repo changelog procedure

### PR DESCRIPTION
**Pull request by codeboten**
Fixes https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11843
Fixes https://github.com/open-telemetry/opentelemetry-collector/issues/5631

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ It is possible that a core approver isn't a contrib approver. In that case, the
 
 1. Make sure the current `main` branch build successfully passes ([Core](https://github.com/open-telemetry/opentelemetry-collector/actions/workflows/build-and-test.yml?query=branch%3Amain) and [Contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/workflows/build-and-test.yml?query=branch%3Amain)).
 
-1. Determine the version number that will be assigned to the release. During the beta phase, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.45.0` as the version to be released, following `v0.44.0`.
+1. Determine the version number that will be assigned to the release. During the beta phase, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.55.0` as the version to be released, following `v0.44.0`.
```
Comment (jpkrohling) [inline docs/release.md:32]:
```suggestion
1. Determine the version number that will be assigned to the release. During the beta phase, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.55.0` as the version to be released, following `v0.54.0`.
```

File: docs/release.md [hunk 2]
```diff
@@ -55,35 +55,36 @@ It is possible that a core approver isn't a contrib approver. In that case, the
...
-### Images and binaries here: https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/v0.45.0
+### Images and binaries here: https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/v0.55.0
 ```
 
 ## Releasing opentelemetry-collector-contrib
 
 1. Prepare Contrib for release.
 
-  * Update CHANGELOG.md file and rename the Unreleased section to the new release name. Add a new unreleased section at top.
+  * Update CHANGELOG.md file, this is now done via `chloggen`. Run the following command from the root of the opentelemetry-collector-contrib repo:
```
Comment (jpkrohling) [inline docs/release.md:72]: Nice!

File: docs/release.md [hunk 3]
```diff
@@ -105,12 +106,12 @@ The last step of the release process creates artifacts for the new version of th
...
 ## Troubleshooting
 
-1. `unknown revision internal/coreinternal/v0.45.0` -- This is typically an indication that there's a dependency on a new module. You can fix it by adding a new `replaces` entry to the `go.mod` for the affected module. 
+1. `unknown revision internal/coreinternal/v0.55.0` -- This is typically an indication that there's a dependency on a new module. You can fix it by adding a new `replaces` entry to the `go.mod` for the affected module. 
 
 ## Release schedule
 
 | Date       | Version | Release manager |
-|------------|---------|-----------------|
+| ---------- | ------- | --------------- |
```
Comment (jpkrohling) [inline docs/release.md:114]: Did you format this automatically using some tool? The tool I use would replace this space with a dash, but if yours format it this way, I'm fine with it.

**Review by djaglowski — APPROVED** [review]

File: docs/release.md (hunk 3 above)
Comment (codeboten) [reply docs/release.md:114]: @jpkrohling yes, its the markdown formatter im using in vscode

**Review by dmitryax — APPROVED** [review]

**Pull request merged**